xds/internal/balancer/clusterimpl/config.go

```diff
@@ -25,32 +25,33 @@ import (
 	"google.golang.org/grpc/serviceconfig"
 )
 
-type dropCategory struct {
+// DropCategory contains the category, and drop ratio.
+type DropCategory struct {
 	Category           string
 	RequestsPerMillion uint32
 }
 
-// lbConfig is the balancer config for weighted_target.
-type lbConfig struct {
-	serviceconfig.LoadBalancingConfig
+// LBConfig is the balancer config for cluster_impl balancer.
+type LBConfig struct {
+	serviceconfig.LoadBalancingConfig `json:"-"`
 
-	Cluster                    string
-	EDSServiceName             string
-	LRSLoadReportingServerName *string
-	MaxConcurrentRequests      *uint32
-	DropCategories             []dropCategory
-	ChildPolicy                *internalserviceconfig.BalancerConfig
+	Cluster                    string                                `json:"cluster,omitempty"`
+	EDSServiceName             string                                `json:"edsServiceName,omitempty"`
+	LRSLoadReportingServerName *string                               `json:"lrsLoadReportingServerName,omitempty"`
```

Comment:
Can this be just `LoadReportingServerName` (with the LRS prefix stripped)?

Reply:
Yes. Done.